src/metabolic_ninja/resources.py

```diff
@@ -185,6 +186,147 @@ def get(self):
         return PRODUCT_LIST
 
 
+class PathwayResource(MethodResource):
+    def get(self, job_id, pathway_id):
```

Comment:
Since there are multiple results per pathway found, please rename this to something else, for example, `row_id`.

```suggestion
    def get(self, job_id, row_id):
```